vm/systemSmartContracts/staking.go

```diff
@@ -450,8 +486,8 @@ func (r *stakingSC) saveStakingData(key []byte, stakedData *StakedDataV2) error
 	return nil
 }
 
-func (r *stakingSC) saveAsStakingDataV1(key []byte, stakedData *StakedDataV2) error {
-	stakedDataV1 := &StakedDataV1{
+func (r *stakingSC) saveAsStakingDataV1P1(key []byte, stakedData *StakedDataV2_0) error {
```

Comment:
maybe move these saveAsFunctions to the back of the file - or to a new file ?

Reply:
done, cleaned-up a little bit auction and staking files.